lib/routemaster/middleware/response_caching.rb

```diff
@@ -34,6 +35,7 @@ def fetch_from_service(env)
             @cache.redis.node_for(namespaced_key).multi do |node|
               node.hset(namespaced_key, body_cache_field(env), response.body)
               node.hset(namespaced_key, headers_cache_field(env), Marshal.dump(response.headers))
+              node.hset(namespaced_key, :event_index, @event_index)
```

Comment:
We should probably convert this sequence of `hset` to an `hmset` to reduce the number of calls.